templates/emailBookingSuccess.html.twig

```diff
@@ -0,0 +1,127 @@
+#}
+
+<!DOCTYPE html>
+<html>
+{% include 'inline-email-head.html.twig' %}
+<body>
+<table role="presentation" cellspacing="0" cellpadding="24" border="0" align="center" class="document">
+    <tr>
+        <td valign="top">
+            <table role="presentation" cellspacing="0" cellpadding="24" border="0" align="center" width="750" class="container" style="background-color: #ffffff;">
+                <tr>
+                    <td>
+                        <table role="presentation" aria-hidden="true" cellspacing="0" cellpadding="0" border="0" align="center" width="100%">
+                            <tr>
+                                <td>
+                                    <table class="top" role="presentation" cellspacing="0" cellpadding="6" border="0" align="right" width="100%">
+                                        <tr>
+                                            <td>
+                                               <div><img align="right" height="50" alt="logo" src="{{ absolute_url(asset('images/logo.png')) }}" ></div>
+                                            </td>
+                                        </tr>
+                                    </table>
+                                    <table class="header" role="presentation" cellspacing="0" cellpadding="6" border="0" align="left" width="100%">
+                                        <tr>
+                                            <td>
+                                                <h2>{{ 'Booking godkendt' }}</h2>
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td>
+                                                {{ 'Kære' }} {{ data.webformSubmission.submissionData.name }}
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td>
+                                                {{ 'Følgende booking er nu registreret i vores system.' }}
+                                            </td>
+                                        </tr>
+                                    </table>
+                                    <table class="booking zebra" cellspacing="0" cellpadding="6" border="0" align="left" width="100%"style="border-bottom: 1px solid #E9ECEF;">
+                                        <tr>
+                                            <td>
+                                                <h3>{{ 'Booking' }}</h3>
+                                            </td>
+                                        </tr>
+                                        <tr class="subject">
+                                            <td>{{ 'Booking emne:' }}</td>
+                                            <td>{{ data.webformSubmission.submissionData.subject }}</td>
```

Comment:
subject is on
```
data.booking.subject
```
instead of through webformSubmission

Reply:
Changed